ArduPlane/quadplane.cpp

```diff
@@ -443,7 +443,6 @@ const AP_Param::GroupInfo QuadPlane::var_info2[] = {
     // @Increment: 1
     // @User: Standard
     AP_GROUPINFO("ACRO_YAW_RATE", 13, QuadPlane, acro_yaw_rate, 90),
-
```

Comment:
accidentally removed line

Reply:
several accidental whitespace changes. I can't get meld to show these for some reason.
Will have to start using "git add -p" religiously :)